dev-embedded/urjtag/urjtag-2021.03.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..12} )
```

Comment:
Did you test this with python 3.12?  For me it doesn't install.

Reply:
My python is ``Python 3.12.0b4 (main, Aug  4 2023, 16:22:12) [GCC 13.2.0] on linux``, 

I tested python very carefully, because I recently debugged another software jtagGUI with python ``3.11.4`` and ``3.12.0b4`` depends on this library.

https://github.com/Najsztub/jtagGUI/pull/5

Reply:
I didn't notice last time, python 3.9 has been removed, so please drop it from PYTHON_COMPAT.

About the error:
```
 * python3_12: running run_in_build_dir installation
Traceback (most recent call last):
  File "/var/tmp/portage/dev-embedded/urjtag-2021.03/work/urjtag-2021.03-python3_12/bindings/python/setup.py", line 3, in <module>
    from distutils.core import setup, Extension
ModuleNotFoundError: No module named 'distutils'
 * ERROR: dev-embedded/urjtag-2021.03::gentoo failed (install phase):
 *   (no error message)
 * 
 * Call stack:
 *     ebuild.sh, line  136:  Called src_install
 *   environment, line 2303:  Called python_foreach_impl 'run_in_build_dir' 'installation'
 *   environment, line 1991:  Called multibuild_foreach_variant '_python_multibuild_wrapper' 'run_in_build_dir' 'installation'
 *   environment, line 1557:  Called _multibuild_run '_python_multibuild_wrapper' 'run_in_build_dir' 'installation'
 *   environment, line 1555:  Called _python_multibuild_wrapper 'run_in_build_dir' 'installation'
 *   environment, line  634:  Called run_in_build_dir 'installation'
 *   environment, line 2279:  Called installation
 *   environment, line 2301:  Called die
 * The specific snippet of code:
 *               "${EPYTHON}" setup.py install --root="${D}" --prefix="${EPREFIX}/usr" || die
 * 
 * If you need support, post the output of `emerge --info '=dev-embedded/urjtag-2021.03::gentoo'`,
 * the complete build log and the output of `emerge -pqv '=dev-embedded/urjtag-2021.03::gentoo'`.
 * The complete build log is located at '/var/tmp/portage/dev-embedded/urjtag-2021.03/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/dev-embedded/urjtag-2021.03/temp/environment'.
 * Working directory: '/var/tmp/portage/dev-embedded/urjtag-2021.03/work/urjtag-2021.03-python3_12/bindings/python'
 * S: '/var/tmp/portage/dev-embedded/urjtag-2021.03/work/urjtag-2021.03'
```
They deprecated distutils and removed it from 3.12, so I don't know why it works for you, probably you have a dependency that is normally missing.  Can you try using distutils-r1 instead of python-r1?  Or maybe asking in #gentoo-python.

Reply:
After your reminder, I compared my local environment, and I found that I forgot to move a local patch. And it does depend on ``dev-python/setuptools`` when building, so it is added as ``BDEPEND``.

Reply:
This is the result of using the latest commit in a clean environment

```bash
running install_scripts
 * python3_11: running python_optimize
 * Optimize Python modules for /usr/lib/python3.11/site-packages
 * Optimize Python modules for /usr/lib/python3.11
 * python3_12: running python_optimize
 * Optimize Python modules for /usr/lib/python3.12/site-packages
 * Optimize Python modules for /usr/lib/python3.12
>>> Completed installing dev-embedded/urjtag-2021.03 into /var/tmp/portage/dev-embedded/urjtag-2021.03/image

 * Final size of build directory: 63096 KiB (61.6 MiB)
 * Final size of installed tree:  11956 KiB (11.6 MiB)

strip: x86_64-pc-linux-gnu-strip --strip-unneeded -N __gentoo_check_ldflags__ -R .comment -R .GCC.command.line -R .note.gnu.gold-version
   /usr/bin/jtag
   /usr/bin/bsdl2jtag
   /usr/lib/python3.11/site-packages/urjtag.cpython-311-x86_64-linux-gnu.so
   /usr/lib64/liburjtag.so.0.0.0
   /usr/lib/python3.12/site-packages/urjtag.cpython-312-x86_64-linux-gnu.so
```

Reply:
The new commit has fixed this issue

Reply:
python3.9 has been droped.